### ISSUE TITLE
[site:statistics] Adding quantity of fields and limit for entities

### ISSUE THREAD

**Issue by cosmicdreams** [hechoendrupal/DrupalConsole#1770]
It would be really great if I could execute a command that can tell me facts about the content that is managed by a Drupal site.  Consider these user stories:

As a developer taking over an existing drupal project, 
I want to know how much content the system has.  
* How many nodes? 
* How many taxonomy terms?  
* How many users?  
* How many different kinds of these things?

As a developer I want to know how many and which fields are associated with a given field-able entity.

Basically these facts are all simple queries that have been done on projects many times before.  Rule of Three dictates that I find a way to automate this.  This tool seems like the best place to put this work.

**Comment by jmolivas** [comment]
@cosmicdreams @darol100 PR https://github.com/hechoendrupal/DrupalConsole/pull/1844 merged.

The command names is `site:statistics` and this is how the output looks like:  
![drupal-statistics](https://cloud.githubusercontent.com/assets/366275/12731434/d2772ee2-c8e5-11e5-8324-67d272c3a4bb.png)

**Comment by cosmicdreams** [comment]
OOOOOOOO nice!

Next phase:

How many fields?
For each entity, list the fields (by name / machine name) that they contain.

**Comment by darol100** [comment]
Assuming that we are adding the field base, we also need to be able to limit by which type of entity is going to be using. For instance, `site:statistics --fields --node:article.` So this can show all the fields and from the article.

**Comment by cosmicdreams** [comment]
As the recent notification points you to, I've created a new issue at #1863 about the feature addition to this.  Right now I just want to find more questions about the content and configuration that this command can answer

**Comment by enzolutions** [comment]
We could close this issue?

**Comment by cosmicdreams** [comment]
Yes
